scss/components/_pause-screen.scss

```diff
@@ -15,5 +15,6 @@
   @extend .oo-center-vertical-horizontal;
   opacity: 0;
   font-size:72px;
+  z-index: -1;
```

Comment:
We've had several issues with `z-index` lately and setting this to the whole pause screen seems problematic. Please find an alternative way to do this.

Reply:
\+